documentation/operators/src/archive/nodes/network-requester-setup.md

```diff
@@ -1,6 +1,10 @@
-# Network Requesters
+# Network Requester
 
-> Nym Network Requester was built in the [building nym](../binaries/building-nym.md) section. If you haven't yet built Nym and want to run the code, go there first.
+```admonish warning
+**This is an archived page for backwards compatibility. The content of this page is not updated since April 19th 2024. Eventually this page will be terminated!**
```

Comment:
same as the other two node archive `admonish warning`s :)

Reply:
done